RecoParticleFlow/PFClusterProducer/plugins/LegacyPFClusterProducer.cc

```diff
@@ -180,12 +186,18 @@ class LegacyPFClusterProducer : public edm::stream::EDProducer<> {
   const edm::ESGetToken<reco::PFClusterParamsHostCollection, JobConfigurationGPURecord> pfClusParamsToken_;
   const edm::EDPutTokenT<reco::PFClusterCollection> legacyPfClustersToken_;
   const edm::EDGetTokenT<reco::PFRecHitCollection> recHitsLabel_;
+  const edm::ESGetToken<HcalPFCuts, HcalPFCutsRcd> hcalCutsToken_;
+  bool cutsFromDB;
+  HcalPFCuts const* paramPF = nullptr;
   // the actual algorithm
   std::unique_ptr<PFCPositionCalculatorBase> positionCalc_;
   std::unique_ptr<PFCPositionCalculatorBase> allCellsPositionCalc_;
 };
 
 void LegacyPFClusterProducer::produce(edm::Event& event, const edm::EventSetup& setup) {
+  if (cutsFromDB) {
+    paramPF = &setup.getData(hcalCutsToken_);
+  }
```

Comment:
This can be a local variable, it doesn't need to be a member variable:
```suggestion
  HcalPFCuts const* paramPF = cutsFromDB_ ? &setup.getData(hcalCutsToken_) : nullptr;
```